nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/TestElasticSearchQuery.java

```diff
@@ -51,6 +51,7 @@
 import org.nuxeo.runtime.test.runner.Deploy;
 import org.nuxeo.runtime.test.runner.Features;
 import org.nuxeo.runtime.test.runner.FeaturesRunner;
+import org.nuxeo.runtime.test.runner.TransactionalFeature;
```

Comment:
Either a non-needed import, either you wanted to refactor the tests to use `TransactionalFeature#nextTransaction()` :)